### ISSUE TITLE
Have a helpful error message when include fails

### ISSUE THREAD

**Issue by pas256** [ansible/ansible#301]
When running:

```
ansible-playbook myplay.yml
```

I received the following output

```
PLAY [all] ****************************

SETUP PHASE ****************************

ok: [123.123.123.58]

ok: [123.123.123.56]

ERROR: unknown
```

Not very helpful, but I am also doing something wrong - I am including an entire playbook within another playbook.

myplay.yml

```

---
- hosts: all
  user: ubuntu
  sudo: True

  tasks:
    - include: myplay2.yml
```

myplay2.yml

```

---
- hosts: all
  user: ubuntu
  sudo: True
  tasks:
  - name: Echo whoami
    action: command /usr/bin/whoami
```

It would be great if the error message when an include fails was more helpful. Add `-D` doesn't produce anymore output, and there is no traceback.

**Comment by mpdehaan** [comment]
-D I know about, though you should still get a TB in this case.

**Comment by pas256** [comment]
Nope, no traceback.

**Comment by mpdehaan** [comment]
This will be fixed by #314, so closing this one.   Have not forgot about it, just easier to maintain a shorter list.
